KsApi/models/lenses/UserCurrencyLenses.swift

```diff
@@ -0,0 +1,10 @@
+import Prelude
+
+extension UserCurrency {
+  public enum lens {
```

Comment:
Can we not just use `keyPath` lenses instead? All you have to do is update `let chosenCurrency: String` to `public private(set) var chosenCurrency: String`